specification/networkfunction/resource-manager/Microsoft.NetworkFunction/stable/2022-08-01/AzureTrafficCollector.json

```diff
@@ -0,0 +1,1063 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-08-01",
+    "title": "Azure Traffic Collector",
```

Comment:
pl address the breaking changes being flagged. After this remove the ARMChangesRequested label for a review with ARM

Reply:
We have changed CollectorPolicy to be a tracked resource instead of a proxy resource as this change was made in our ARM manifest. The breaking change RemovedEnumValue is occurring as a result of that. As the bot mentioned we need to create a new API version, we created the new version 2022-08-01, and then changed the default api version under that folder to 2022-08-01, which is causing the DefaultValueChanged error. (If this is not needed, we can change it back). Currently our service is only in private-preview and will be public previewed soon, so customers will not be impacted by this change.